build/README.md

```diff
@@ -19,60 +19,49 @@ and allows reading/writing files and resolving Dart source code.
 An interface into the dart [analyzer][pub:analyzer] to allow resolution of code
 that needs static analysis and/or code generation.
 
-## Differences between the `build` package and `pub` + `barback`.
-
-If you currently implement transformers with `package:barback` for use with
-Dart v1 `pub build` and `pub serve`, see [Upgrading from barback][].
-
 ## Implementing your own Builders
 
-If you have written a `barback` `Transformer` in the past, then the
-[`Builder`][dartdoc:Builder] API should be familiar to you. The main difference
-is that `Builders` must always configure outputs based on input extensions.
+A `Builder` gets invoked one by one on it's inputs, and may output new files.
 
 The basic API looks like this:
 
 ```dart
 abstract class Builder {
-  /// You can only output files in `build` that are configured here. You are not
-  /// required to output all of these files, but no other [Builder] is allowed
-  /// to produce the same outputs.
+  /// You can only output files that are configured here. You are not required
+  /// to output all of these files, but no other builder is allowed to produce
+  /// the same outputs.
```

Comment:
Should we expand this comment to better describe what this actually means (suffix substitution)?

Reply:
I expanded it a bit - I don't want to go into too much detail here.